src/async/control-flow/select.md

```diff
@@ -74,4 +74,7 @@ async fn main() {
   pass `&mut future` instead of the future itself, but this can lead to
   issues, further discussed in the pinning slide.
 
+* Unmatched branches are dropped, which cancels their futures (unless a reference
+  was used).
```

Comment:
This is what I was trying to say in the previous bullet point, but a bit clearer. Perhaps the two could be merged?

Reply:
I tried and ended up with two bullet points, but ... I think it's improved.